### ISSUE TITLE
Backup with folder data

### ISSUE THREAD

**Issue by secuvera** [secuvera/SpotMyBackup#1]
Current Backups ignore folders. It would be nice if the were handled by the backup. From
https://community.spotify.com/t5/user/viewprofilepage/user-id/82541

**Comment by rave14210** [comment]
Yes, I am really hoping to have this feature too! Otherwise my playlists are all over the place...
Thank you for the great tool!

Ramon

**Comment by rave14210** [comment]
Oh yes, I am really hoping to have this feature too! Otherwise my playlists are all over the place...
Thank you for the great tool!
